sql/2021/compression/compression_formats.sql

```diff
@@ -0,0 +1,16 @@
+#standardSQL
+  # compression_formats.sql : What compression formats are being used (gzip, brotli, etc)
+SELECT
+  client,
+  resp_content_encoding,
+  COUNT(0) AS num_requests,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
```

Comment:
Nit: also include the total for completeness

```suggestion
  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
```